src/Workspace/index.jsx

```diff
@@ -188,13 +189,20 @@ class Workspace extends React.Component {
 
   render() {
     const terminateButton = (
-      <Button
-        className='workspace__button'
-        onClick={this.handleTerminateButtonClick}
-        label='Terminate Workspace'
-        buttonType='primary'
-        isPending={this.state.notebookStatus === 'Terminating'}
-      />
+      //wrap up termate button with Popconfirm
+      <Popconfirm
+        title="Are you sure to terminate your workspce?"
+        onConfirm={this.handleTerminateButtonClick}
+        okText="Yes"
```

Comment:
```suggestion
        okText='Yes'
```
portal uses `prefer-single` for `jsx-quotes` rule